### PR TITLE
feat: add network faults example between block node and consensus node with code cleanup and docs

### PR DESCRIPTION
**Pull request by leninmehedy**
## Description

- Added example network faults scenario between block-node and consenus node
- Added documentations
- Grouped chaos for different components such as `block-node` and consensus-node for maintainability

### Related Issues

* Closes #
* Related to #

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `go.mod` and/or `go.sum` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit
  message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a
   breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any
   type. NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                       |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: dev/taskfile/Taskfile.root.yml [hunk 1]
```diff
@@ -244,6 +243,57 @@ tasks:
...
+      - |
+        kubectl get pods -n "{{.NAMESPACE}}" -o json | \
+          jq -r '.items[] | select(.metadata.labels["app.kubernetes.io/name"] | strings | startswith("block-node")) | .metadata.labels["app.kubernetes.io/instance"]' | \
+          sort -u | \
+          while read -r instance; do
+            echo "  Labelling $instance"
+            kubectl label pods -n "{{.NAMESPACE}}" \
+              -l "app.kubernetes.io/instance=$instance" \
+              solo.hedera.com/type=block-node \
+              solo.hedera.com/region=ap \
```
Comment (brunodam) [inline dev/taskfile/Taskfile.root.yml:279]: have we documented that the region is hardcoded to ap? Should we? Or should we make it flexible.

**Review by brunodam — APPROVED** [review]
LGTM... just one minor comment

File: dev/taskfile/Taskfile.root.yml (hunk 1 above)
Comment (leninmehedy) [reply dev/taskfile/Taskfile.root.yml:279]: It's just an example code. For now the example sets to `ap`. I anticipate that we shall need add more in subsequent PRs.
